CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.0)
+project(symbiflow-arch-defs)
+
+include(make/util.cmake)
```

Comment:
nit: Should this directory be called `cmake` now?

Reply:
We'll leave the old Make system in place until everything is working, so I'm fine leaving it as make for now.  I don't think anyone is gonna be confused.